source/utilities.tex

```diff
@@ -19238,7 +19238,6 @@
 % FIXME: For now, keep the format grammar productions out of the index, since
 % they conflict with the main grammar.
 % Consider renaming these en masse (to fmt-* ?) to avoid this problem.
-\newcommand{\fmtnontermdef}[1]{{\BnfNontermshape#1\itcorr}\textnormal{:}}
```

Comment:
I would prefer if you would leave the uses of `\fmtnontermdef` as-is and just change its definition to be `\ninontermdef`.
(This keeps the semantic markup of the fmt-related grammar.)